src/lae_site/handlers/create_subscription.py

```diff
@@ -84,73 +76,73 @@ def handle_stripe_create_customer_errors(self, trace_back, error, details, email
         send_plain_email('info@leastauthority.com', 'support@leastauthority.com', body, headers)
         raise RenderErrorDetailsForBrowser(details)
 
-    def create_customer(self, stripe_authorization_token, user_email):
+    def create_customer(self, stripe_authorization_token, user_email, request):
         try:
             return self._stripe.create(
                 authorization_token=stripe_authorization_token,
                 plan_id=PLAN_ID,
                 email=user_email,
             )
         except stripe.CardError as e:
+            # Always return 402 on card errors
+            request.setResponseCode(402)
             # Errors we expect: https://stripe.com/docs/api#errors
             note = "Note: This error could be caused by insufficient funds, or other charge-disabling "+\
                 "factors related to the User's payment credential.\n"
             self.handle_stripe_create_customer_errors(
-                traceback.format_exc(100), e,
-                details=e.message,
-                email_subject="Stripe Card error ({})".format(user_email),
-                notes=note,
+              traceback.format_exc(100), e,
+              details=e.message.encode("utf-8"),
+              email_subject="Stripe Card error ({})".format(user_email),
+              notes=note,
             )
         except stripe.APIError as e:
+            # Should return the same as Authentication error
+            request.setResponseCode(401)
             self.handle_stripe_create_customer_errors(
-                traceback.format_exc(100), e,
-                details=(
-                    "Our payment processor is temporarily unavailable, "
-                    "please try again in a few moments."
-                ),
-                email_subject="Stripe API error ({})".format(user_email),
+              traceback.format_exc(100), e,
+              details="Our payment processor is temporarily unavailable,"+
+                  " please try again in\ a few moments.",
```

Comment:
You can rely on compile-time literal string concatenation here.

    (
        "foo"
        "bar"
    )

is the same as

    (
         "foo" +
         "bar"
    )

except that the former is evaluated at compile time and the latter at runtime.